_init/_editConfig.sh

```diff
@@ -1,4 +1,37 @@
 #!/bin/bash
+_setConfigSettings() {
+    # Setting config options
+    rm /tmp/cci_config_opts #delete temp file if it already exists.
+    mktemp /tmp/cci_config_opts
+    if [[ -n $ORBCONFIGOPT_PRCOMMENT ]]
+    then
+      echo "          add-pr-comment: true" >> /tmp/cci_config_opts
+      echo "You have enabled the add-pr-comment option. This requires a bot user."
+      echo
+      read -p "Enter the bot GitHub User Token. This will be added to your project under the GH_BOT_USER_TOKEN environment variable.  :" -r GH_BOT_USER_TOKEN
+      curl -X POST --header "Content-Type: application/json" -d '{"name":"GH_BOT_USER_TOKEN", "value":"'"$GH_BOT_USER_TOKEN"'"}' "https://circleci.com/api/v1.1/project/:vcs-type/"$CCI_ORGANIZATION"/"$CCI_REPO"/envvar?circle-token=:"$CCI_TOKEN""
```

Comment:
It looks like the GH_BOT_USER_TOKEN environment variable didn't get created when I ran this because of `:vcs-type` and `:"$CCI_TOKEN`